OneCommunityGlobal/HighestGoodNetworkApp
# Kaixiang create 24/48/72hr btn

### Description:
1. Created 24/48/72 hours button for users to display time logs on Tasks tab.
Ex: If the user click 24hours button on Feb 18, it will fetch time log from Feb 17 - Feb 18 (Feb 18 included).
2. Time logs will be displayed under each user.  
3. Created Hide Timelog button to hide displayed time logs.

### Mainly Changes Explained:
1. Front end UI of buttons.
2. Restructure team member tasks code, extract reusable component to make it more cleaner.
3. Created fetch time logs for period functions.
![new_buttons](https://user-images.githubusercontent.com/98206107/219904435-d50f418f-dc6b-4fec-91e9-9b3cb56d6b5f.JPG)
![Display_time_entries](https://user-images.githubusercontent.com/98206107/219904437-1d4f3b41-f257-4f59-be9e-4e61b38dc08f.JPG)


### How to Test:
1. Log time entries, edit date from three days ago to today's date.
5. Click these three buttons to see if time entries can be displayed correctly.
6. Use Hide Timelog button to hide displaying time logs.
